src/test/javascript/spec/component/lecture/lecture-import.component.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { NgbHighlight, NgbPagination } from '@ng-bootstrap/ng-bootstrap';
+import { ButtonComponent } from 'app/shared/components/button.component';
+import { SortService } from 'app/shared/service/sort.service';
+import { PageableSearch, SearchResult, SortingOrder } from 'app/shared/table/pageable-table';
+import { MockComponent, MockDirective, MockPipe } from 'ng-mocks';
+import { of } from 'rxjs';
+import { ArtemisTestModule } from '../../test.module';
+import { SortByDirective } from 'app/shared/sort/sort-by.directive';
+import { SortDirective } from 'app/shared/sort/sort.directive';
+import { Lecture } from 'app/entities/lecture.model';
+import { LecturePagingService } from 'app/lecture/lecture-paging.service';
+import { LectureImportComponent, TableColumn } from 'app/lecture/lecture-import.component';
+
+describe('LectureImportComponent', () => {
+    let fixture: ComponentFixture<LectureImportComponent>;
+    let comp: LectureImportComponent;
+    let pagingService: LecturePagingService;
+    let sortService: SortService;
+    let searchForLecturesStub: jest.SpyInstance;
+    let sortByPropertyStub: jest.SpyInstance;
+    let searchResult: SearchResult<Lecture>;
+    let state: PageableSearch;
+    let lecture: Lecture;
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule, FormsModule],
+            declarations: [
+                LectureImportComponent,
+                MockComponent(ButtonComponent),
+                MockComponent(NgbHighlight),
+                MockComponent(NgbPagination),
+                MockDirective(SortByDirective),
+                MockDirective(SortDirective),
+            ],
+            providers: [],
+            schemas: [],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(LectureImportComponent);
+                comp = fixture.componentInstance;
+                pagingService = TestBed.inject(LecturePagingService);
+                sortService = TestBed.inject(SortService);
+                searchForLecturesStub = jest.spyOn(pagingService, 'searchForLectures');
+                sortByPropertyStub = jest.spyOn(sortService, 'sortByProperty');
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', () => {
+        fixture.detectChanges();
+        expect(LectureImportComponent).not.toBeNull();
+    });
```

Comment:
Tests like this are, altough some of them exist in our codebase, useless. Please remove it :)